_editorial_policies/04_publication_ethics.md

```diff
@@ -0,0 +1,51 @@
+- Unpublished information, arguments, or interpretations disclosed in a submitted manuscript should not be used in an editor’s own research except with the consent of the author. However, if such information indicates that some of the editor’s own research is unlikely to be profitable, the editor could ethically discontinue the work. When a manuscript is so closely related to the current or past research of an editor as to create a conflict of interest, the editor should arrange for some other qualified editor to take editorial responsibility for that manuscript. In some cases, it may be appropriate to tell an author about the editor’s research and plans in that area.
+- If an editor is presented with convincing evidence that the main substance or conclusions of a report published in an editor’s journal are erroneous, the editor should facilitate publication of an appropriate report pointing out the error.  This error should be corrected in the current GitHub version of the article, and a note may be placed on the original article indicating this correction. 
+- An author may request that the editor not use certain reviewers in consideration of a manuscript. However, the editor may decide to use one or more of these reviewers, if the editor feels their opinions are important in the fair consideration of a manuscript. This might be the case, for example, when a manuscript seriously disagrees with the previous work of a potential reviewer
+
+### Ethical Obligations of Authors
+
+- Authors are expected to adhere to the following ethical guidelines; infractions may result in the application of sanctions by the editor(s), including but not limited to the suspension or revocation of publishing privileges.
+- An author’s central obligation is to present an accurate and complete account of material presented, absolutely avoiding deception, including the data collected or used. In the case of presenting new data (which will generally not be the case in  LiveCoMS articles, but is not prohibited), the procedures, software links, parameters and the data provided should contain sufficient detail and reference to public sources of information to permit a trained professional to reproduce the observations. 
+- Authors should provide data and methods to other researchers, and are encouraged to submit their data to a public database, where available. 					
+- An author is obligated to cite those publications that have been influential in preparing their materials and that will guide the reader quickly to the earlier work that is essential for understanding the present investigation. 
+- In submitting a manuscript for publication, an author should inform the editor of related manuscripts that the author has under editorial consideration or in press. Copies of those manuscripts should be supplied to the editor, and the relationships of such manuscripts to the one submitted should be indicated.					
```

Comment:
Is this one relevant to us? We're not focused on original research.

Reply:
Well, I think it's relevant if there are overlaps in material covered.

Reply:
It increases the burden on editors; if I have to just assign a manuscript for review, I can do it promptly. If I have to read the manuscript and two or three other related manuscripts to decide whether they are related and how much, that might take me a couple of weeks to get to. And that's even before assigning reviewers. Then I have to decide if this material should also go to the reviewers and if I should get them to assess it. I see it lengthening the review process considerably, and we haven't given any guidance on how this would factor in to the editorial decision/review process. It will likely bubble up to the managing editors and we'll have to debate how we should handle it. If that's what we want that's fine, but I just don't think it should be needed.

Reply:
How about just inform the editors? Then we can discuss it at a higher level, without worry about the details.